index.html

```diff
@@ -39,17 +39,142 @@ <h3 class="connect">LET'S CONNECT</h3>
+        <div class="project-description">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <div>
+          <button class="languages">HTML</button>
+          <button class="languages">CSS</button>
+          <button class="languages">JAVASCRIPT</button>
+        </div>
+        <div>
+          <button class="see-project">See Project</Button>
+        </div>
+      </div>
+
+      <div class="projects">
+        <img src="./images/Snapshoot Portfolio (1).png" alt="">
+        <h2 class="works-title">TONIC</h2>
+        <p><span class="canopy">Canopy</span> <img src="./images/Counter.png" alt="Counter"> <span class="dev"> End Dev
+            <img src="./images/Counter.png" alt="Counter"> 2015</span></p>
+        <div class="project-description">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <div>
+          <button class="languages">HTML</button>
+          <button class="languages">CSS</button>
+          <button class="languages">JAVASCRIPT</button>
+        </div>
+        <div>
+          <button class="see-project">See Project</Button>
+        </div>
+      </div>
+
+      <div class="projects">
+        <img src="./images/Snapshoot Portfolio (2).png" alt="">
+        <h2 class="works-title">TONIC</h2>
+        <p><span class="canopy">Canopy</span> <img src="./images/Counter.png" alt="Counter"> <span class="dev"> End Dev
+            <img src="./images/Counter.png" alt="Counter"> 2015</span></p>
+        <div class="project-description">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <div>
+          <button class="languages">HTML</button>
+          <button class="languages">CSS</button>
+          <button class="languages">JAVASCRIPT</button>
+        </div>
+        <div>
+          <button class="see-project">See Project</Button>
+        </div>
+      </div>
+
+      <div class="projects">
+        <img src="./images/Snapshoot Portfolio(3).png" alt="">
+        <h2 class="works-title">TONIC</h2>
+        <p><span class="canopy">Canopy</span> <img src="./images/Counter.png" alt="Counter"> <span class="dev"> End Dev
+            <img src="./images/Counter.png" alt="Counter"> 2015</span></p>
+        <div class="project-description">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <div>
+          <button class="languages">HTML</button>
+          <button class="languages">CSS</button>
+          <button class="languages">JAVASCRIPT</button>
+        </div>
+        <div>
+          <button class="see-project">See Project</Button>
+        </div>
+      </div>
+
+      <section class="about">
+        <div>
+          <div class="first-heading">
+            <h1>About Myself</h1>

```

Comment:
- [ Required ] A good webpage usually has only one `h1` tag which marks the most important `heading` on your webpage. In this project, this `h1` tag is in the introduction section. To make your webpage semantically correct I would like to suggest that you change this `h1` tag in the  _About_ section to any of the following `hx` tags `h2`, `h3`, `h4`, `h5` or `h6` tags.